config/rm/settings.ini

```diff
@@ -45,6 +45,9 @@ pa.rm.execute.script.timeout=180000
 # All other selection scripts will be rejected.
 pa.rm.select.script.authorized.dir=
 
+# The pa.rm.select.script.authorized.dir is browsed every refreshperiod time to load authorized scripts.
+pa.rm.select.script.authorized.dir.refreshperiod=60000
```

Comment:
Because the rm holds and controls the execution of selection scripts. Thus, if we want to restrict both scheduler scripts and rm scripts, we need two different configurations.